tests/helpers/index.ts

```diff
@@ -0,0 +1,49 @@
+import { format } from 'url'
+import { stringify } from 'querystring'
+
+import nock from 'nock'
+
+import {
+  API_PROTOCOL,
+  API_HOST,
+  API_PATH,
+  API_VALID_RESPONSE,
+} from '../../src/constants'
+
+import { applyDefaultOptionValues, getPrivateHeaders } from '../../src/internal'
+
+const mockUrl = format({
+  protocol: API_PROTOCOL,
+  hostname: API_HOST,
+})
+
+export const defaultResponseMock = {
+  error: API_VALID_RESPONSE,
+  result: true,
+}
+
+export const mockCredentials = {
+  key: 'mockKey',
+  secret: 'mockSecret',
+}
+
+export const mockResolveCallback = (scope, done) =>
+  jest.fn((error, data) => {
+    expect(error).toBe(null)
+    expect(data).toBe(true)
+    expect(scope.isDone()).toBeTruthy()
+    return done()
+  })
+
+export const prepareNock = (

```

Comment:
this is a very ambiguous name, as it does not say anything about which endpoint for what get mocked